eventually/vote/views.py

```diff
@@ -353,3 +367,27 @@ def delete(self, request, team_id, event_id, vote_id, answer_id=None):  # pylint
 
         Answer.delete_by_id(answer_id)
         return RESPONSE_200_DELETED
+
+def extended_answers(request, event_id, vote_id):
```

Comment:
Rename this method.  for example `get_answers_with_member`.
And make it a class AnswerView method.